src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/impl/CurrentAssetsTransformerImpl.java

```diff
@@ -0,0 +1,195 @@
+package uk.gov.companieshouse.web.accounts.transformer.smallfull.impl;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.accounts.smallfull.BalanceSheetApi;
+import uk.gov.companieshouse.api.model.accounts.smallfull.CurrentAssetsApi;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+import uk.gov.companieshouse.web.accounts.model.smallfull.CashAtBankAndInHand;
+import uk.gov.companieshouse.web.accounts.model.smallfull.CurrentAssets;
+import uk.gov.companieshouse.web.accounts.model.smallfull.Debtors;
+import uk.gov.companieshouse.web.accounts.model.smallfull.Stocks;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.Transformer;
+
+import java.util.Objects;
+import java.util.stream.Stream;
+
+@Component("currentAssetsTransformer")
+public class CurrentAssetsTransformerImpl implements Transformer {
+
+    @Override
+    public void addCurrentPeriodToApiModel(BalanceSheetApi balanceSheetApi, BalanceSheet balanceSheet) {
+
+        if (hasCurrentPeriodCurrentAssets(balanceSheet)) {
+            CurrentAssetsApi currentAssetsApi = new CurrentAssetsApi();
+            currentAssetsApi.setStocks(balanceSheet.getCurrentAssets().getStocks().getCurrentAmount());
```

Comment:
@marcransome , just wondering.shouldn't we check if the values are not null before getting them? Otherwise, we could end up with null pointer exception, couldn't we?